src/bindings/js/node/tests/model.test.js

```diff
@@ -28,3 +28,24 @@ describe('Node.js Model.isDynamic()', () => {
     assert.strictEqual(model.isDynamic(), expectedStatus, 'Expected isDynamic to return false for a static model');
   });
 });
+
+describe('Node.js Model.get_output_shape()', () => {
+  
+    it('should return the shape of the output tensor', () => {
+      const outputShape = model.get_output_shape(0);
+      assert.ok(Array.isArray(outputShape), 'get_output_shape() should return an array');
+      assert.strictEqual(outputShape.length, 2, 'Expected output shape to have 2 dimensions');
+    });
+  
+    it('should throw an error if the output index is out of range', () => {
+      assert.throws(() => {
+        model.get_output_shape(1);
+      }, /^Error: Invalid output index: 1\.$/, 'Expected get_output_shape to throw an error when called with an invalid output index');
```

Comment:
Looks like it doesn't work, because error message is different in https://github.com/openvinotoolkit/openvino/pull/23843/files#diff-4147b940e5a212756fe408dac0ca4593b3ddec09a27deaba95895035d7aee4dcR143